tests/scripts/generate_test_keys.py

```diff
@@ -89,6 +89,32 @@ def get_look_up_table_entry(key_type: str, group_id_or_keybits: str,
     yield "      {0}, sizeof({0}),\n".format(priv_array_name)
     yield "      {0}, sizeof({0}) }},".format(pub_array_name)
 
+
+def write_output_file(output_file_name: str, arrays: List[str], look_up_table: List[str]):
+    with open(output_file_name, 'wt') as output:
+        output.write("""
```

Comment:
Minor: there's now a blank line at the beginning of the file. You can avoid this with a backslash:
```suggestion
        output.write("""\
```